### PR TITLE
LPD-39837 Enhancements on the display menu, on Apps Page, Solutions Page, app Details Page

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master-private/css.css [hunk 1]
```diff
@@ -4,10 +4,56 @@
...
+.lfr-nav-item.nav-item.selected.active .nav-link {
+	border-bottom: #004AD7 solid 3px !important;
+	color: #0053F0 !important;
+	font-weight:600;
+}
+
+.lfr-nav-item.nav-item.dropdown:has(.child-menu.dropdown-menu .active.selected) .nav-link {
+	border-bottom: #004AD7 solid 3px !important;
+	color: #0053F0 !important;
+	font-weight:600;
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master-private/css.css:39]:
```suggestion
	font-weight: 600;
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master-private/css.css [hunk 2]
```diff
@@ -4,10 +4,56 @@
...
+
+.lfr-nav-item.nav-item:hover {
+	background-color: #ffffff !important;
+	font-weight: none !important;
+}
+
+.lfr-nav-item.nav-item.selected.active .nav-link {
+	border-bottom: #004AD7 solid 3px !important;
+	color: #0053F0 !important;
+	font-weight:600;
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master-private/css.css:33]:
```suggestion
	font-weight: 600;
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master-private/css.css [hunk 3]
```diff
@@ -4,10 +4,56 @@
...
+}
+
+.child-menu.dropdown-menu {
+	border-radius: 8px !important;
+	padding: 0px !important;
+}
+
+.card.main-image {
+	border-radius:10px !important;
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master-private/css.css:48]:
```suggestion
	border-radius: 10px !important;
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master/css.css [hunk 4]
```diff
@@ -4,10 +4,56 @@
...
+
+.lfr-nav-item.nav-item:hover {
+	background-color: #ffffff !important;
+	font-weight: none !important;
+}
+
+.lfr-nav-item.nav-item.selected.active .nav-link {
+	border-bottom: #004AD7 solid 3px !important;
+	color: #0053F0 !important;
+	font-weight:600;
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master/css.css:33]:
```suggestion
	font-weight: 600;
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master/css.css [hunk 5]
```diff
@@ -4,10 +4,56 @@
...
+}
+
+.lfr-nav-item.nav-item.dropdown:has(.child-menu.dropdown-menu .active.selected) .nav-link {
+	border-bottom: #004AD7 solid 3px !important;
+	color: #0053F0 !important;
+	font-weight:600;
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master/css.css:39]:
```suggestion
	font-weight: 600;
```

**Review by kevenleone — CHANGES_REQUESTED** [review]

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master/css.css [hunk 6]
```diff
@@ -4,10 +4,56 @@
...
+
+.product-gallery .arrow.next {
+	border-bottom-right-radius: 10px;
+	border-top-right-radius: 10px;
+}
+
+.product-gallery .arrow.prev {
+	border-bottom-left-radius: 10px;
+	border-top-left-radius: 10px;
+}
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master/css.css:59]: @welitonpe could you please try to sort?

Doesn't need to be perfect, but as much as possible

```suggestion
.card.main-image {
	border-radius: 10px !important;
}

.child-menu.dropdown-menu {
	border-radius: 8px !important;
	padding: 0px !important;
}

.lfr-nav-item.nav-item {
	border-radius: 8px;
	margin-right: 0.5rem;
	transition: all 0.3s ease-in-out;
}

.lfr-nav-item.nav-item .nav-link {
	border-bottom: transparent solid 3px !important;
}

.lfr-nav-item.nav-item:hover {
	background-color: #ffffff !important;
	font-weight: none !important;
}

.lfr-nav-item.nav-item.selected.active .nav-link {
	border-bottom: #004AD7 solid 3px !important;
	color: #0053F0 !important;
	font-weight: 600;
}

.lfr-nav-item.nav-item.dropdown:has(.child-menu.dropdown-menu .active.selected) .nav-link {
	border-bottom: #004AD7 solid 3px !important;
	color: #0053F0 !important;
	font-weight: 600;
}

.product-gallery .arrow.next {
	border-bottom-right-radius: 10px;
	border-top-right-radius: 10px;
}

.product-gallery .arrow.prev {
	border-bottom-left-radius: 10px;
	border-top-left-radius: 10px;
}
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master-private/css.css [hunk 7]
```diff
@@ -4,10 +4,56 @@
...
+
+.product-gallery .arrow.next {
+	border-bottom-right-radius: 10px;
+	border-top-right-radius: 10px;
+}
+
+.product-gallery .arrow.prev {
+	border-bottom-left-radius: 10px;
+	border-top-left-radius: 10px;
+}
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master-private/css.css:59]:
```suggestion
.card.main-image {
	border-radius: 10px !important;
}

.child-menu.dropdown-menu {
	border-radius: 8px !important;
	padding: 0px !important;
}

.lfr-nav-item.nav-item {
	border-radius: 8px;
	margin-right: 0.5rem;
	transition: all 0.3s ease-in-out;
}

.lfr-nav-item.nav-item .nav-link {
	border-bottom: transparent solid 3px !important;
}

.lfr-nav-item.nav-item:hover {
	background-color: #ffffff !important;
	font-weight: none !important;
}

.lfr-nav-item.nav-item.selected.active .nav-link {
	border-bottom: #004AD7 solid 3px !important;
	color: #0053F0 !important;
	font-weight: 600;
}

.lfr-nav-item.nav-item.dropdown:has(.child-menu.dropdown-menu .active.selected) .nav-link {
	border-bottom: #004AD7 solid 3px !important;
	color: #0053F0 !important;
	font-weight: 600;
}

.product-gallery .arrow.next {
	border-bottom-right-radius: 10px;
	border-top-right-radius: 10px;
}

.product-gallery .arrow.prev {
	border-bottom-left-radius: 10px;
	border-top-left-radius: 10px;
}
```

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Review by kevenleone — APPROVED** [review]

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**